### ISSUE TITLE
DOTA v1 Reproducibility Issue

### ISSUE THREAD

**Issue by Nikhil-Kumar-Patel** [ultralytics/ultralytics#20298]
Hi, thanks for the great work on YOLOv11!

I trained the `yolov11n-obb` model on the DOTA v1 dataset (multi-scale version) using the following configuration:

- Task: OBB
- Model: yolov11n-obb.yaml
- Dataset: DOTAv1.yaml
- Epochs: 100
- Image size: 640
- Batch size: 16
- Pretrained: true
- Augmentation: Auto augment set to `randaugment`
- Other transforms: `mosaic=1.0`, `mixup=0.0`, `copy_paste=0.0`, `erasing=0.4`
- Val split: `val`
- Training deterministic: true
- Optimizer and LR settings: `lr0=0.01`, `lrf=0.01`, `momentum=0.937`, `weight_decay=0.0005`, `warmup_epochs=3.0`

Despite matching the setup closely with the default recommendations, I did not reach the reported performance metrics for this model on the DOTA dataset. I would appreciate any help in identifying what might be missing or incorrect in my configuration.

Some possible areas I'm unsure about:
- Is `randaugment` the same policy used for the reported results?
- Any specific tricks used for small object detection that aren’t obvious in the default settings?
- Was `imgsz=640` sufficient for OBB detection on DOTA, or was a larger resolution used?

Thanks in advance for the guidance! Please let me know if any additional logs or results would help.

<img width="992" alt="Image" src="https://github.com/user-attachments/assets/de72bd89-9298-4187-99f5-1c72f3ca4d62" />

**Comment by UltralyticsAssistant** [comment]
👋 Hello @Nikhil-Kumar-Patel, thank you for your detailed report and for using YOLO11 with the DOTA v1 dataset 🚀! We appreciate your thorough configuration breakdown—this really helps us understand your workflow.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) (MRE) including your exact training command, relevant YAML files, and any logs or results you have observed. This will assist us in pinpointing any reproducibility issues or differences.

If this is a custom training ❓ Question, please include as much detail as possible, such as example images from your dataset and full training logs. Also, make sure you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/), as OBB detection on DOTA can be sensitive to several factors.

You can also browse our [Docs](https://docs.ultralytics.com/) for more information, including [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples.

Join the Ultralytics community where it suits you best:
- For real-time chat: [Discord](https://discord.com/invite/ultralytics) 🎧
- For in-depth discussions: [Discourse](https://community.ultralytics.com/)
- For sharing with peers: [Subreddit](https://reddit.com/r/Ultralytics)

## Upgrade

Upgrade to the latest `ultralytics` package and ensure all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) are installed in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify whether your issue persists in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO can be run in up-to-date verified environments (all dependencies included: [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

This is an automated response to help you get started—an Ultralytics engineer will review your report and assist you soon! 🛠️
